frontend/src/components/Pages/Media/Media.css

```diff
@@ -40,6 +40,25 @@ a.nostyle:link {
    height:190px !important
 }
 
+@media all and (min-width:0px) and (max-width: 480px) {
+  /* put your css styles in here */
+  .newsbox {
+    height: 400px;
+    width: 400px;
```

Comment:
avoid using hard coded height and width in responsive app. use max-height max-width instead.